frontend/src/app/spot/components/text-field/text-field.component.ts

```diff
@@ -46,6 +46,10 @@ export class SpotTextFieldComponent implements ControlValueAccessor {
 
   @Input() public value = '';
 
+  @Input() public pattern:string|undefined;
+
+  @Input() public inputmode:string = 'text';
```

Comment:
Could you add doc comments to both of these? If you merge `release/12.3`, you should see the others that are now there.

Reply:
Done 👍